docs/src/Table.doc.js

```diff
@@ -288,6 +288,181 @@ card(
   />
 );
 
+card(<Card name="Table.RowExpandable" />);
+
+card(
+  <PropTable
+    showHeading={false}
+    // $FlowIssue[prop-missing]
+    Component={Table?.Row}
+    props={[
+      {
+        name: 'children',
+        type: 'React.Node',
+      },
+      {
+        name: 'expandedContents',
+        type: 'React.Node',
+      },
+      {
+        name: 'hoverStyle',
+        type: '"none" | "gray"',
+        defaultValue: 'gray',
+      },
+    ]}
+  />
+);
+
+card(
+  <Example
+    name="Example: Table Row Expanable"
+    defaultCode={`
+<Table>
+  <Table.Header>
+    <Table.Row>
+    <Table.HeaderCell>
+  <Box display="visuallyHidden">
+    <Text weight="bold">Open/Close row</Text>
+  </Box>
+</Table.HeaderCell>
+      <Table.HeaderCell>
+        <Text weight="bold">Name</Text>
+      </Table.HeaderCell>
+      <Table.HeaderCell>
+        <Text weight="bold">House</Text>
+      </Table.HeaderCell>
+      <Table.HeaderCell>
+        <Text weight="bold">Birthday</Text>
+      </Table.HeaderCell>
+    </Table.Row>
+  </Table.Header>
+  <Table.Body>
+    <Table.RowExpandable
+    expandedContents=<Box maxWidth={236} padding={2} column={12}>
```

Comment:
@faisalgedi could we also use the curly bracket here?

```
expandedContents={<
```
instead of
```
expandedContents=<
```

Also recommend copy/pasting the example in a separate file + running prettier on it